src/references/game_events.md

```diff
@@ -375,7 +382,7 @@ function CheckEvent()
 end
 
 --Required to stop the task and remove event object once the event is no longer active
-function CleanupEvent()
+local function CleanupEvent()
```

Comment:
Function needs to be above the `CheckEvent`. When declaring local functions, they need to be ordered correctly.